<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_renderingadvancedgroup</class>
 <widget class="QWidget" name="toolbar_renderingadvancedgroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rendering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_billboardEnable">
     <property name="text">
      <string>Round points</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_billboardFeather">
     <property name="text">
      <string>Feather</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="slider_billboardFeather">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_billboardFeather">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line_billboard">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_edlEnable">
     <property name="text">
      <string>Eye-dome lighting</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_edlStrength">
     <property name="text">
      <string>Strength</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="slider_edlStrength">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_edlStrength">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_edlRadius">
     <property name="text">
      <string>Radius (px)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="slider_edlRadius">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_edlRadius">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>6.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_edlBias">
     <property name="text">
      <string>Bias</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="slider_edlBias">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_edlBias">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
